partuniverse/partsmanagement/models.py

```diff
@@ -754,7 +754,7 @@ class VerifiedStock(models.Model):
         User,
         verbose_name=_("Created by"),
         help_text=_("The user which verified the stock."),
-        on_delete=models.PROTECT,
+        on_delete=models.DO_NOTHING,
```

Comment:
Why not SET_NULL, too?

Reply:
Is that the better solution? Should I update the PR?

Reply:
Yes, I think so. 
If the user is gone that was done the verified stock report, than there is no need to keep that information. Also this might lead to some inconsistent data.